src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cn from 'classnames';
+import { TodoFilterMethod } from '../../types/Todo';
+
+type Props = {
+  currentMethod: TodoFilterMethod;
+  onSelect: (method: TodoFilterMethod) => void;
+};
+
+const Filter: React.FC<Props> = ({ currentMethod, onSelect }: Props) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      {Object.entries(TodoFilterMethod)
+        .filter(([key]) => key !== 'Default')
```

Comment:
i'd recommend to do this kind of computations before return in the component body for more readability


```suggestion
      const filteredMethods = Object.entries(TodoFilterMethod)
        .filter(([key]) => key !== 'Default');
        
      return (... 
      {filteredMethods.map(...)
      )
```